src/components/capture/creation/DetailsForm.tsx

```diff
@@ -138,10 +125,26 @@ function NewCaptureDetails(props: NewCaptureDetailsProps) {
                             }
                         }}
                     />
-                ) : (
+                ) : isError ? (
+                    error
+                ) : schema.properties.image.oneOf.length < 0 ? (
                     <Alert severity="warning">
                         <FormattedMessage id="captureCreation.missingConnectors" />
+                        {schema.properties.image.oneOf.length}
                     </Alert>
+                ) : (
+                    <>
+                        <Skeleton
+                            variant="rectangular"
+                            height={40}
+                            width="50%"
+                        />
+                        <Skeleton
+                            variant="rectangular"
+                            height={40}
+                            width="50%"
+                        />
+                    </>
                 )}
```

Comment:
The fall through "default" is loading since our first check is if post processing is done.